custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -45,6 +48,9 @@ class WorkAnIssue(
         if (random.random.nextFloat() < commentProbability) {
             comment(loadedIssuePage)
         }
+        if (random.random.nextFloat() < attachScreenShotProbability) {
```

Comment:
I'd extract a method for that. Like this: https://github.com/atlassian/jces-1209/blob/fd1fd2d4da880bf2dfb5328338377b0efc1dd5ba/custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt#L34-L39